nerfstudio/models/splatfacto.py

```diff
@@ -734,7 +738,11 @@ def get_outputs(self, camera: Cameras) -> Dict[str, Union[torch.Tensor, List]]:
             tile_bounds,
         )  # type: ignore
         if (self.radii).sum() == 0:
-            return {"rgb": background.repeat(int(camera.height.item()), int(camera.width.item()), 1)}
+            return {
+                "rgb": background.repeat(int(camera.height.item()), int(camera.width.item()), 1),
+                "depth": torch.ones(int(camera.height.item()), int(camera.width.item()), 1) * 10,
```

Comment:
also do we need to worry about the device that these are on? maybe we should do `background.new_ones()`, `background.new_zeros()` to make sure it's matched?